unetsocket/c/Makefile

```diff
@@ -1,14 +1,18 @@
 CC = gcc
 CFLAGS += -std=c99 -Wall -Wextra -Werror -Wfloat-equal -Wconversion -Wparentheses -pedantic -Wunused-parameter -Wunused-variable -Wreturn-type -Wno-unused-function -Wredundant-decls -Wreturn-type -Wunused-value -Wswitch-default -Wuninitialized -Winit-self -O2
-# FJAGE_VER=c8e62495f2283d1b57a559112ccce3ccd608bb2a
-FJAGE_VER=master
+FJAGE_VER=7275142c2bc846d2f07400d7d628f213cb5b56ab
 FJAGE_DIR=fjage-$(FJAGE_VER)
 
+CONTRIB_VER = bd95ed77ef7c90edbba04dd67f8fb235262e08c8
```

Comment:
We don't need `CONTRIB_VER` anymore right?

Reply:
I have removed it now while minor cleanup. Yet to commit.